static/app/views/automations/components/actionFilters/ageComparison.tsx

```diff
@@ -84,7 +92,21 @@ function TimeField() {
       options={TIME_CHOICES}
       onChange={(option: SelectValue<TimeUnit>) => {
         onUpdate({comparison: {...condition.comparison, time: option.value}});
+        removeError(condition.id);
       }}
     />
   );
 }
+
+export function validateAgeComparisonCondition(
```

Comment:
It's a little strange for us to be exporting util functions from a component file. Maybe it would make more sense to put these utils in a "validation" file or folder instead? Although as I'm writing this I do kind of understand how it's nice to colocate all the field logic together.